Cargo.toml

```diff
@@ -9,7 +9,7 @@ homepage = "https://github.com/pyo3/pyo3"
 repository = "https://github.com/pyo3/pyo3"
 documentation = "https://docs.rs/crate/pyo3/"
 categories = ["api-bindings", "development-tools::ffi"]
-license = "Apache-2.0"
+license = "MIT/Apache-2.0"
```

Comment:
@DataTriny I think the correct SPDX conformant way is `MIT OR Apache-2.0`? Most tooling like [CycloneDX](https://github.com/CycloneDX/cyclonedx-rust-cargo) will parse the slash variant, but warn about it.

Please also add a news fragment describing the license change as documented in [the guide](https://pyo3.rs/v0.18.3/contributing#documenting-changes).